manual/core/compression/README.md

```diff
@@ -19,8 +19,13 @@ Compression must be set before opening a session, it cannot be changed at runtim
 
 
 Two algorithms are supported out of the box: [LZ4](https://github.com/jpountz/lz4-java) and
-[Snappy](http://google.github.io/snappy/). Both rely on third-party libraries, declared by the
-driver as *optional* dependencies; if you enable compression, you need to explicitly depend on the
+[Snappy](http://google.github.io/snappy/). The LZ4 implementation is a good
+first choice; it offers fallback implementations in case native libraries fail
+to load and
+[benchmarks](http://java-performance.info/performance-general-compression/)
```

Comment:
I have one concern about that link. I opened it multiple times (5 times) and 2 out of it I got `error establishing a connection`. I don't know if we want to add a link to an unreliable page?

Reply:
I'm getting this error too.